ParameterMetaDataBackup.xml

```diff
@@ -19721,5 +19721,11 @@
       <Range>1 255</Range>
       <User>Advanced</User>
     </CMD_TOTAL>
+    <AOA_CRIT>

```

Comment:
you have places this under ardutracker, there are multiple vechile class's wityhin this file.
normally I generate this off latest stable also. so can be skipped in the commit